.github/workflows/docker-push-to-registries.yml

```
@@ -195,17 +221,41 @@ jobs:
       - name: Build, tag, and push image to authenticated docker registry
         id: build-images
         uses: docker/build-push-action@v6.9.0
+        if: ${{ !inputs.DRY_RUN }}
         with:
           builder: ${{ steps.buildx.outputs.name }}
           platforms: linux/amd64,linux/arm64
           # Dependabot triggered PRs don't have credentials to push images
-          push: ${{ github.actor != 'dependabot[bot]' }}
+          push: ${{ github.actor != 'dependabot[bot]' && !inputs.DRY_RUN }}
+          context: ${{ inputs.PATH }}
+          file: ${{ inputs.PATH }}/${{ inputs.DOCKERFILE }} # NOTE: relative path (inputs.PATH defaults to '.')
+          tags: ${{ steps.meta.outputs.tags }}
+          target: ${{ inputs.TARGET }}
+          build-args: |
+            ${{ inputs.BUILD_ARGS }}
+      - name: Build and tag image
```

Can we name this step more clearly e.g.

```suggestion
      - name: Build Docker Tarball
```